src/js/angular/core/directives/rdf-resource-search.directive.js

```diff
@@ -0,0 +1,66 @@
+angular
```

Comment:
You can put this directive and its template in its own package under `core/directives`.